src/app/modules/ix-forms/components/ix-permissions/ix-permissions.component.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../../../../../node_modules/text-security/text-security.css';
+
+:host {
+  color: var(--fg1);
+  display: block;
+
+  margin-bottom: 12px;
+  margin-top: 18px;
+  padding: 8px 0;
+}
+
+td {
+  padding: 5px 7px;
+}
+
+.tooltip {
+  margin-left: 8px;
+
+  vertical-align: -4px;
+}
+
+.permissions-container {
+  &.disabled {
+    opacity: 0.38;
+
+    mat-checkbox {
+      cursor: not-allowed;
+    }
+  }
+
+  &.readonly {
+    input {
+      cursor: copy;
+    }
+  }
+}
+
+.label-container {
+  margin-bottom: 1px;
+  margin-left: 1px;
+}
+
+.label {
+  font-size: x-small;
```

Comment:
it makes all labels smaller than others in a form:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/351613/158541424-72da31b6-60b5-4ade-9fc4-0525088d5eb2.png">
<img width="260" alt="image" src="https://user-images.githubusercontent.com/351613/158541451-5d51e307-f3e7-4119-9c7d-4282062487da.png">

Reply:
It looks consistent with the other inputs on the new ix-forms.
![image](https://user-images.githubusercontent.com/24204846/158694027-2281f99d-dba7-4c7b-a78e-945aff455d5f.png)

Reply:
![Screenshot_40](https://user-images.githubusercontent.com/22980553/158766181-ba8cebc0-dbaa-4d64-ad06-ed86181eec92.png)

Looks label is small . I think it's better to make same font-size with above checkbox label `Show console Messages` considering new type of input.